### PR TITLE
add oc expose -h explanation on generator

### PR DESCRIPTION
**Pull request by juanvallejo**
Related bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1420165

cc @openshift/cli-review @kargakis

### CODE REVIEW

**Comment by juanvallejo** [comment]
[test]

File: docs/man/man1/oc-expose.1 [hunk 1]
```diff
@@ -50,7 +50,7 @@ There is also the ability to expose a deployment configuration, replication cont
 
 .PP
 \fB\-\-generator\fP=""
-    The name of the API generator to use.
+    The name of the API generator to use. Defaults to "route/v1". Available generators include "service/v1", "service/v2", and "route/v1". "service/v1" will automatically name the port "default", while "service/v1" will leave it unnamed.
```
Comment (fabianofranz) [inline docs/man/man1/oc-expose.1]: `"service/v1" will (...), while "service/v1"` ;)

Comment (juanvallejo) [inline docs/man/man1/oc-expose.1]: Ah, thanks for catching that :) Fixed

**Comment by fabianofranz** [comment]
[merge]

**Pull request merged**